csvcubed/csvcubed/utils/validators/uri.py

```diff
@@ -23,6 +23,25 @@ class SomeModel(PydanticModel):
 
             _some_uri_validator = validate_uri("some_uri")
     """
+
     return validator(attr_name, allow_reuse=True, always=not is_optional)(
         ensure_looks_like_uri
     )
+
+    
+def validate_uris(attr_name: str, is_optional: bool = False) -> classmethod:
+    """
+    pydantic validator to ensure that an attribute has a string value which looks like a URI.
```

Comment:
Again, update this comment, and possibly change the name of the function to make it clear it applies to attributes which hold lists of strings.